app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt

```diff
@@ -243,6 +245,13 @@ class PasswordFragment : Fragment(R.layout.password_recycler_view) {
         try {
             listener = object : OnFragmentInteractionListener {
                 override fun onFragmentInteraction(item: PasswordItem) {
+                    if (settings.getString(PreferenceKeys.SORT_ORDER) == PasswordRepository.PasswordSortOrder.RECENTLY_USED.name) {
+                        //save the time when password was used
+                        val editor = settings.edit()
+                        editor.putString(item.file.absolutePath, System.currentTimeMillis().toString())
```

Comment:
Please use a separate shared preferences file here as well as the Kotlin `edit` extension.

Reply:
I'll do it